src/index.ts

```diff
@@ -101,6 +101,33 @@ const getScalarDefinition = (value: ScalarDefinition | ScalarGeneratorName): Sca
     return value;
 };
 
+const getCustomScalarValue = (customScalar: ScalarDefinition, opts: Options<NamedTypeNode>) => {
+    // If there is a mapping to a `casual` type, then use it and make sure
+    // to call it if it's a function
+    const embeddedGenerator = casual[customScalar.generator];
```

Comment:
I think this feature should also allow for customer `faker` generator functions, I know at least that I would use that features! This is especially necessary if the plan is still to deprecate `casual` in favor of `faker`

Reply:
Yes indeed, I think we should do a separate PR to support faker, as it's not directly related

Reply:
Here is the issue
https://github.com/ardeois/graphql-codegen-typescript-mock-data/issues/97